requirements/prod-requirements.in

```diff
@@ -10,6 +10,9 @@ ipython==7.16.1
 ndg-httpsclient
 pyasn1
 
+# SSO
```

Comment:
this is going to go back into regular requirements, which sucks because people will need to install `libxmlsec1`. I know the author of that article you linked didn't like this library, but there is no clear alternative and I'm not about to write my own library from scratch with zero crypto experience.